tchannel/tornado/dispatch.py

```diff
@@ -63,13 +63,13 @@ def my_method(request, response, proxy):
 
     FALLBACK = object()
 
-    def __init__(self):
+    def __init__(self, _handler_returns_response=False):
         self.handlers = defaultdict(lambda: Handler(
             self.not_found, RawSerializer(), RawSerializer())
         )
+        self._handler_returns_response = _handler_returns_response
```

Comment:
@abhinav luckily I was able to get this whole thing done with just this flag - unfortunately I still needed a flag :(